tutor/test_t2_05_AnalyzeCollegeWorkflow.py

```diff
@@ -36,17 +39,23 @@ def setUp(self):
         """Pretest settings."""
         self.ps = PastaSauce()
         self.desired_capabilities['name'] = self.id()
-        self.Teacher = Teacher(
-            use_env_vars=True,
-            pasta_user=self.ps,
-            capabilities=self.desired_capabilities
-        )
+        # self.Teacher = Teacher(
```

Comment:
Remove the excess Users, especially the ones with defined login credentials; re-include the PastaSauce object in the user parameters (it's how SauceLabs is initiated as well as the target for post-test results).